README.md

```diff
@@ -72,10 +72,14 @@ But what does it do?
 
 Works with OSCC `v1.2.1` and up.
 
-### Dependencies
+### Linux Dependencies
 
 - `python3` (`sudo apt install python3`)
-- `pip3` (`sudo apt install python3-pip`)
```

Comment:
Are there any dependencies on imported things like the color terminal output that need a pip install?